docs/admin.rst

```diff
@@ -437,15 +437,6 @@ VERSION_CODENAME is "focal"
 
   sudo unattended-upgrades
 
-
-VERSION_CODENAME is "xenial"
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code:: sh
-
-  sudo cron-apt -i -s
-
-
```

Comment:
I think by 2.10 or 2.2.0 we can remove the OS platform check here from the docs entirely, but fine to keep for now to avoid any confusion.